crates/sncast/src/main.rs

```diff
@@ -122,9 +124,22 @@ fn main() -> Result<()> {
     let runtime = Runtime::new().expect("Failed to instantiate Runtime");
 
     if let Commands::Script(script) = cli.command {
+        let manifest_path = match cli.path_to_scarb_toml {
+            Some(path) => path,
+            None => get_scarb_manifest().context("Failed to obtain manifest path from Scarb")?,
+        };
+        let metadata = get_scarb_metadata_with_deps(&manifest_path)?;
+        let package_metadata = get_package_metadata(&metadata, &manifest_path)?;
+        let mut artifacts = build(&BuildConfig {
+            scarb_toml_path: manifest_path.clone(),
+            json: cli.json,
```

Comment:
What passing cli.json to scarb implies?

Reply:
scarb output will be in json format, just like sncast. this is for consistency reasons (it's reasonable to assume that users might want the whole output in json, not just some parts of it)